File: output/gemma4-e2b/avatar/Java/Python/atcoder_ABC149_B.py
import sys

def solve():
    # Read A, B, K from a single line of input
    try:
        line = sys.stdin.readline()
        if not line:
            return
        A, B, K = map(int, line.split())
    except EOFError:
        return
    except ValueError:
        # Handle case where input might be malformed, though unlikely in CP context
        return

    # Implementation of calc()
    x = A
    y = B
    x = A - K

    if x < 0:
        y = B + x
        x = 0
        if y < 0:
            y = 0

    # Implementation of showResult()
    print(x, y)

solve()

File: output/gemma4-e2b/avatar/Python/Java/atcoder_ABC149_B.java
import java.util.Scanner;
import java.lang.Math;

public class atcoder_ABC149_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read the input line and parse cookies
        String line = scanner.nextLine();
        String[] parts = line.split(" ");
        
        // Assuming input has at least 3 elements based on the Python indexing
        if (parts.length < 3) {
            // Handle error or insufficient input if necessary, though usually assumed valid in competitive programming context
            return;
        }
        
        int[] cookies = new int[parts.length];
        for (int i = 0; i < parts.length; i++) {
            cookies[i] = Integer.parseInt(parts[i]);
        }

        int c0 = cookies[0];
        int c1 = cookies[1];
        int c2 = cookies[2];
        
        // leftOver = cookies[0] - cookies[2]
        int leftOver = c0 - c2;
        
        // takahashi = max(0, leftOver)
        int takahashi = Math.max(0, leftOver);
        
        int result;
        
        // Conditional expression: (str(cookies[1]) if